Run Project Implementation Initial Commits

kevin-german opened the pull request:
Initial upload of files to handle implementation for File Upload UI.

## Describe your changes
Created files for Run Project UI
## Issue link and documentation link
[💻Issue](https://github.com/PowerSynth/front-end/issues/14)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added the necessary documentation.

kevin-german committed:
Run Project Implementation Initial Commits

Initial upload of files to handle implementation for File Upload UI.

kevin-german committed:
Merge branch 'main' into 14-run-project-implementation

connorielliott commented on src/components/WelcomePage.css:
```diff
@@ -24,3 +24,5 @@
   cursor: pointer;
   width: 200px;
 }
+
+
```
Possibly undo this change to keep git history clean?

connorielliott reviewed (dismissed):
Possibly just address my comment, other then that changes look great and the upload file is very nice looking 👍

kevin-german committed:
Update WelcomePage.css

Updated to resolve comment.

kevin-german committed:
Update WelcomePage.css

kevin-german committed:
Merge branch 'main' into 14-run-project-implementation

Signed-off-by: Kevin German <98424956+kevin-german@users.noreply.github.com>

kevin-german committed:
Run Project Implementation Updates

-Moved server.js file into backend folder
-Added popup components to display file(s) user has uploaded, and allow for the user to remove the file(s), which is done by making calls to server.js file
-Added buttons for canceling 'Run Project' (can also click off of popup window to hide it) and running project after file(s) have been uploaded (functionality to still be implemented)
-UI Changes

kevin-german committed:
Merge branch 'main' into 14-run-project-implementation

restyled-io[bot] committed:
Restyle Run Project Implementation Initial Commits (#36)

* Restyled by clang-format

* Restyled by prettier

* Restyled by whitespace

---------

Co-authored-by: Restyled.io <commits@restyled.io>

kevin-german committed:
Imported BluePrintJS and made resulting UI changes

-Imported BlueprintJS
-Made changes to UI to make use of Blueprint library
-By using the library, I was able to restructure/eliminate some of the components I had previously created to implement the popup

restyled-io[bot] committed:
Restyle Run Project Implementation Initial Commits (#41)

* Restyled by clang-format

* Restyled by prettier

* Restyled by whitespace

---------

Co-authored-by: Restyled.io <commits@restyled.io>

kevin-german committed:
Merge branch 'main' into 14-run-project-implementation

kevin-german committed:
Update package-lock.json

Commit to fix restyling error

restyled-io[bot] committed:
Restyle Run Project Implementation Initial Commits (#46)

* Restyled by clang-format

* Restyled by jq

* Restyled by prettier

---------

Co-authored-by: Restyled.io <commits@restyled.io>

atejedag reviewed (approved):
Looks good
